openapi.yaml

```
@@ -2165,6 +2176,631 @@ paths:
       security:
         - oscal_auth:
             - read:parties
+  /plan-of-action-and-milestones:
+    get:
+      tags:
+        - OSCAL Plan of Action and Milestone
+      summary: Returns all OSCAL plan of action and milestones
```

I feel like `Plans of action and milestones` sounds better as the plural here. At the very least, we should be consistent on this. 

```suggestion
      summary: Returns all OSCAL plans of action and milestones
```